rotkehlchen/chain/ethereum/oracles/uniswap.py

```diff
@@ -295,7 +300,27 @@ def get_pool(
             ] for fee in (3000, 500, 10000)],
         )
 
-        return [pool[0] for pool in result]
+        # get liquidity for each pool and choose the pool with the highest liquidity
+        best_pool, max_liquidity = to_checksum_address(result[0][0]), 0
+        for query in result:
+            if query[0] == ZERO_ADDRESS:
+                continue
+            pool_address = to_checksum_address(query[0])
+            pool_contract = EthereumContract(
+                address=pool_address,
+                abi=UNISWAP_V3_POOL_ABI,
+                deployed_block=12369621,  # Factory deployment block
```

Comment:
make this into a constant with an appropriate name

Reply:
Is not possible to make it a constant since the `pool_address` is variable here

Reply:
You market all the EthereumContract, okey will change this